boards/posix/native_posix/native_posix_64.yaml

```diff
@@ -14,4 +15,8 @@ supported:
   - netif:eth
   - usb_device
   - adc
+  - i2c
+  - spi
   - gpio
+testing:
```

Comment:
when should not enable this as a default platform without a good reason. This just duplicate tests and increases CI times for no apparent good reason. 64 bit is tested in other platforms already.

Reply:
In my specific case, using `native_posix` was not an option, and 64-bit worked for my use case. I understand the concern about CI times, but allowing it to be used doesn't mean it must be built and tested every time.

The `native_posix_64` is closer to what most developers use daily as a development host, so it also seems a good choice.